packages/electron/README.md

```diff
@@ -35,6 +35,22 @@ export default {
 }
 ```
 
+you can use `process.env.VITE_DEV_SERVER_URL` when the vite command is called 'serve'
+
+```js
+// electron main.js
+const win = new BrowserWindow({
+  title: 'Main window',
+})
+
+if (process.env.VITE_DEV_SERVER_URL) {
```

Comment:
Still need wizard spelling in `electron main.ts`
https://github.com/vitejs/vite/pull/8843#issuecomment-1173207397

Reply:
But Vite3 supported to use `process.env.XXX` in `build.lib` mode.
https://github.com/vitejs/vite/blob/c1fa219f7da09d8fca0140c79e62dbce6d54b0e5/packages/vite/src/node/plugins/define.ts#L13-L31